moi/__init__.py

```diff
@@ -16,6 +16,24 @@
 from moi.context import Context  # noqa
 
 
+def _support_directory():
+    """Get the path of the support_files directory"""
+    from os.path import join, dirname, abspath
```

Comment:
Not sure I follow, why would code executing on an ipengine need to
reference the .js? There isn't anything here that prevents that either
On May 29, 2015 8:27 AM, "Antonio Gonzalez" <notifications@github.com>
wrote:

> In moi/__init__.py
> <https://github.com/biocore/mustached-octo-ironman/pull/28#discussion_r31334823>
> :
>
> > @@ -16,6 +16,24 @@
> >  from moi.context import Context  # noqa
> >
> >
> > +def _support_directory():
> > +    """Get the path of the support_files directory"""
> > +    from os.path import join, dirname, abspath
>
> Isn't this running on a job so you have to do the imports within it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/mustached-octo-ironman/pull/28/files#r31334823>
> .
>